go.mod

```diff
@@ -43,8 +43,8 @@ require (
 )
 
 require (
-	github.com/google/go-github/v62 v62.0.0
-	github.com/google/go-github/v62 v62.0.0
```

Comment:
I only changed this: https://github.com/kairos-io/kairos-agent/pull/396/files#diff-3b4dce18e3a84018e27c76dc16cfb623754d23871ad2e90ef4abdf5ef2a1692dL26-R26

and run `go mod tidy` :shrug:

`go mod` is weird :D